network/dial_queue_test.go

```diff
@@ -46,3 +50,111 @@ func TestDialQueue(t *testing.T) {
 		t.Fatal("timeout")
 	}
 }
+
+func TestDel(t *testing.T) {
+	type task struct {
+		id     string
+		action string // "add"/"delete"/"pop"
```

Comment:
I would take this 1 step further and make sure these values: `add`, `delete` and `pop` are constants (in a sense enums), so whenever we reference them in code, we know exactly which value we're hitting

Reply:
I tried but there will be a cyclic dependency (because `ReadLibp2pKey` is defined in network package)

Reply:
I see 🧐

What about declaring it within the test file?

Reply:
Fixed in https://github.com/0xPolygon/polygon-sdk/pull/156/commits/99afa50917c33cb3dd8d8d4d705af45cc56a7776